src/pages/App.tsx

```diff
@@ -99,9 +101,9 @@ export default function App() {
           <Polling />
           <ErrorBoundary fallback={<p>An unexpected error occured on this part of the page. Please reload.</p>}>
             <Switch>
-              <Route path="/">
-                <Earn />
-              </Route>
+              <Route exact strict path="/zap" component={Zap} />
```

Comment:
nit: strict is bit unnecessary here? if someone types revo.market/zap/, the url would return 404

Reply:
i have a default router so instead of a 404 you will be directed to the /zap page :) buuuut indeed `pool/` will be routed to `/zap`...this is actually what ubeswap has, i think it is safer than an inexact match in case we have a url like `/pools` in the future